### ISSUE TITLE
Remove creation of linux,stdout-package in prom_init

### ISSUE THREAD

**Issue by mpe** [linuxppc/linux#125]
We have code in `prom_init.c` which creates a `linux,stdout-package` property:

```
		prom_setprop(prom.chosen, "/chosen", "linux,stdout-package",
			     &val, sizeof(val));
```

This is a non-standard property, and seems to have never even been used in mainline.

It was added in 2004:

https://github.com/mpe/linux-fullhistory/commit/689fe5072fe9a0dec914bfa4fa60aed1e54563e6

And then the only use, which was already inside `#if 0` was removed a month later:

https://github.com/mpe/linux-fullhistory/commit/1fbe5a6d90f6cd4ea610737ef488719d1a875de7

**Comment by mopsfelder** [comment]
I've sent a patch to fix this issue:

https://lkml.kernel.org/r/20180712171904.18971-1-muriloo@linux.ibm.com

**Comment by mopsfelder** [comment]
Patch applied to powerpc next:

https://git.kernel.org/pub/scm/linux/kernel/git/powerpc/linux.git/commit/?h=next&id=ec9336396a98f61290f45e8bb942458a1b2f299b

**Comment by mpe** [comment]
Fixed in ec9336396a98f61290f45e8bb942458a1b2f299b, thanks @mopsfelder :champagne:
